apps/api/README.md

```diff
@@ -0,0 +1,19 @@
+# IrvineHacks Site API
+
+This "app" contains the source code for the backend API used by the site.
+
+## Local Development
+
+For local development, run
+
+```shell
+python src/dev.py
+```
+
+which will start a Uvicorn server with auto-reload.
+
+## Deployment Configuration
+
+For deployment, the following environment variables need to be set:
+
+-   `PYTHONPATH=src/api` to properly import Python modules
```

Comment:
Question: I remember last year, you created a `.env` file with this key value pair in it. Is there a reason you're not doing that this time around?

Reply:
Good catch, looks like I missed committing it because the name was in `.gitignore`.
The reason to have it locally is mainly for the VS Code Python extension.